internal/api/hcpopenshiftcluster.go

```diff
@@ -200,6 +201,20 @@ func (cluster *HCPOpenShiftCluster) validateVersion() []arm.CloudErrorBody {
 		})
 	}
 
+	versionID := cluster.Properties.Version.ID
+	if versionID != "" {
+		// Check if version is in X.Y format and append ".0" to convert X.Y to X.Y.Z
+		if matched, _ := regexp.MatchString(`^\d+\.\d+$`, versionID); matched {
+			cluster.Properties.Version.ID = versionID + ".0"
+		} else {
```

Comment:
Won't this trigger if the control plane version has been set to X.Y.Z previously?

Reply:
for the august release, we only want the customer to be able to specify x.y for the control planes version. We'll have CS implement x.y later but for now, cs can only take x.y.z. This is a temporary solution